frontend/src/metabase/visualizations/click-actions/Mode/Mode.ts

```diff
@@ -37,37 +37,31 @@ export class Mode {
     settings: Record<string, any>,
     extraData?: Record<string, any>,
   ): ClickAction[] {
-    // keeping try/catch here as MBQL was failing silently
```

Comment:
Reverts changes from the original PR

Reply:
well, maybe it's not needed, but mbql was failing silently without it